packages/fx-core/resource/package.nls.json

```diff
@@ -395,6 +395,7 @@
   "error.aad.manifest.OptionalClaimsIsMissing": "optionalClaims is missing\n",
   "error.aad.manifest.OptionalClaimsMissingIdtypClaim": "optionalClaims access token doesn't contain idtyp claim\n",
   "error.aad.manifest.AADManifestIssues": "AAD manifest has following issues that may potentially break the Teams App:\n",
+  "error.aad.manifest.DeleteOrUpdatePermissionFailed": "Failed to update or delete an existing permission when it's enabled. One possible reason is the ACCESS_AS_USER_PERMISSION_ID environment variable is changed for selected environment. Ensure your permission id(s) are identical with the actual AAD application and try again.\n",
```

Comment:
Failed to -> Unable to

One possible reason is the -> One possible reason is that the

Reply:
thanks, updated.